samcli/lib/init/__init__.py

```diff
@@ -145,3 +150,30 @@ def _enable_application_insights(application_insights: bool, output_dir: str, na
         template_modifier = ApplicationInsightsTemplateModifier(template_file_path)
         template_modifier.modify_template()
         EventTracker.track_event(EventName.USED_FEATURE.value, UsedFeature.INIT_WITH_APPLICATION_INSIGHTS.value)
+
+
+def _create_default_samconfig(package_type: str, output_dir: str, name: str) -> None:
+    """
+    Init post-processing function to create a default samconfig.toml
+    file and place it into the cookie cutter project.
```

Comment:
I am confused, are we generating the toml file in the generated sam app project or in the cookie cutter project ?

Reply:
we generate a file and add it to the generated project, post cookie-cutter.